tests/workgroup/CMakeLists.txt

```diff
@@ -141,6 +141,10 @@ add_test_pocl(NAME "workgroup/range_md_large_grid"
               EXPECTED_OUTPUT ""
               COMMAND "run_kernel" "range_md.cl" "1000" "128" "1" "1")
 
+add_test_pocl(NAME "workgroup/ballot"
```

Comment:
Like Tapio mentioned, ballot.cl is missing, and also please add this to one of the `set_tests_properties(... LABELS "cpu;workgroup")` code blocks.